common/script/public/userServices.js

```diff
@@ -107,10 +107,14 @@ angular.module('habitrpg')
                       if (err) {
                         var message = err.code ? err.message : err;
                         console.log(message);
-                        if (MOBILE_APP) Notification.push({type:'text',text:message});
-                        else Notification.text(message);
-                        // In the case of 200s, they're friendly alert messages like "Your pet has hatched!" - still send the op
-                        if ((err.code && err.code >= 400) || !err.code) return;
+                        if (typeof message === 'string') 
+                          message = [message];  
+                        _.each(message, (msg) => {
```

Comment:
Ah, that didn't crop up when I ran the tests.  Is there some way I can catch these kinds of issues?